Index: c0915/HelloWorld.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- c0915/HelloWorld.java	(revision )
+++ c0915/HelloWorld.java	(revision )
@@ -0,0 +1,7 @@
+package c0915;
+
+public class HelloWorld {
+    static public void main(String[] arg){
+        System.out.println("hello, world!");
+    }
+}
